frontend/src/Student/components/ShowCard.js

```diff
@@ -30,13 +76,28 @@ const ShowCard = (props) => (
       </Col>
     </Row>
     <hr />
-    <Row style={{ minHeight: '250px' }}>
-      <Col className='text-center align-self-center justify-content-center h-100'>
-        <Link to={`/submit?to=${props.show.id}`} className='d-block w-100 h-100'>
+    <Row style={{ minHeight: '250px' }} className='align-items-center'>
+      <Col
```

Comment:
Shouldn't the submissions be dynamically swapped out with the new submission button rather than having it in the show card piece itself?

Reply:
Could you clarify? I'm confused what you mean.

Reply:
The issue was resolved in kayla's pr so nevermind